spreadsheet/line-chart-from-csv/main.go

```diff
@@ -0,0 +1,129 @@
+// Copyright 2024 FoxyUtils ehf. All rights reserved.
+package main
+
+import (
+	"encoding/csv"
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/unidoc/unioffice/common/license"
+	"github.com/unidoc/unioffice/spreadsheet"
+)
+
+func init() {
+	// Make sure to load your metered License API key prior to using the library.
+	// If you need a key, you can sign up and create a free one at https://cloud.unidoc.io
+	err := license.SetMeteredKey(os.Getenv(`UNIDOC_LICENSE_API_KEY`))
+	if err != nil {
+		panic(err)
+	}
+}
+
+func main() {
+	args := os.Args
+	if len(args) < 2 {
+		fmt.Printf("Usage %s go run main.go input.csv output.xlsx", os.Args[0])
+		return
+	}
+
+	inputPath := args[1]
+	outputPath := args[2]
+
+	sliceData, err := readCsv(inputPath)
+	if err != nil {
+		log.Fatalf("error reading csv: %s", err)
+	}
+
+	kmSlice := strings.Split(sliceData[1][0], ";")
+	dateSlice := strings.Split(sliceData[2][0], ";")
+
+	ss := spreadsheet.New()
+	defer ss.Close()
+	sheet := ss.AddSheet()
+
+	// Create all of our data
+	row := sheet.AddRow()
+	row.AddCell().SetString("Date")
+	row.AddCell().SetString("Length")
+
+	startRowNumber := row.RowNumber() + 1
+	endRowNumber := startRowNumber + 1
+	for r := 1; r < len(dateSlice); r++ {
+		km, err := strconv.Atoi(kmSlice[r])
+		if err != nil {
+			panic(err)
```

Comment:
Maybe log error instead of panic?

Reply:
updated at b124e68ff732566d430e6aad84cdc974d3d5ef3d